### ISSUE TITLE
The way to apply attention mask.

### ISSUE THREAD

**Issue by lzyhha** [HiDream-ai/HiDream-I1#67]
Hello, in [https://github.com/HiDream-ai/HiDream-I1/blob/main/hi_diffusers/models/attention_processor.py#L71](https://github.com/HiDream-ai/HiDream-I1/blob/main/hi_diffusers/models/attention_processor.py#L71), I notice that the attention mask is applied by multiplying mask and key. 
Can you please explain why not fill -inf to the attention like [pytorch](https://docs.pytorch.org/docs/stable/generated/torch.nn.functional.scaled_dot_product_attention.html)? 

Thanks.
